administrator/components/com_localise/Field/TranslationsField.php

```diff
@@ -157,4 +244,33 @@ protected function getGroups()
 
 		return $groups;
 	}
+
+	/**
+	 * Method to add a suffix to an array.
+	 *
+	 * @param   array   $array   An array of core files.
+	 * @param   string  $suffix  The suffix to add to each file.
+	 *
+	 * @return  array   The modified array
+	 */
+	public static function suffix_array_values($array, $suffix = '')
+	{
+		if (!is_array($array))
+		{
+			return false;
+		}
+
+		// Suffix the values and respect the keys
```

Comment:
`Suffix the values` means what?

Reply:
> `Suffix the values` means what?

// Suffix the values and respect the keys

That comment was already rhere.
Means transform com_localise to com_localise.ini or com_localise.sys to com_localise.sys.ini if the suffix from the function is "ini"
All that fine names are stored within an array, so, the function apply that to all matches within the array.

Job time, i will revise all when come back at home.

Reply:
I suggest therefore
`Add a suffix to the value...`